guideline/cfh.md

```diff
@@ -89,7 +89,7 @@ off by the PAC before they can be put in place.
 2. The on-site team must be geographically located in a country in the Asia
    Pacific and Australasia region. 
 
-3. The on-site team must be willing to actively coordinate with the PAC board
```

Comment:
PAC == PyCon APAC Committee.  Changing it to APAC looks wrong.  But we might change "PAC board" to "PAC".